### ISSUE TITLE
Migrate cronjobs to Argo CronWorkflows

### ISSUE THREAD

**Issue by pacospace** [thoth-station/thoth-application#654]
**Is your feature request related to a problem? Please describe.**

- [ ] deploy workflow-controller to namespaces hosting cronworkflows

Migrate cronjob in Thoth to CronWorkflow:

- [ ] increase quota on thoth-infra
- [ ] move all the -job to thoth-infra namespace
- [ ] deploy Argo's workflow-controller to thoth-infra
- [ ] graph-update-job
- [ ] graph-refresh-job
- [ ] package-release-job
- [ ] cve-update-job
- [ ] graph-backup
- [ ] slo-reporter
- [ ] advise-reporter

**Describe the solution you'd like**
Use Argo Cronworkflow for flexibility, monitoring and extensibility: https://github.com/thoth-station/thoth-application/issues/429#issuecomment-727820282

**Describe alternatives you've considered**

**Additional context**

**Comment by harshad16** [comment]
constraint: this would require workflow-controller to be deployed in other namespaces as well.

**Comment by VannTen** [comment]
> constraint: this would require workflow-controller to be deployed in other namespaces as well.

Is a cluster-scoped controller not possible as an alternative ?

**Comment by harshad16** [comment]
As we work on shared cluster, cluster-scoped controller are most of the time hard to get deployed.

**Comment by VannTen** [comment]
> As we work on shared cluster, cluster-scoped controller are most of the time hard to get deployed.

Could operate-first offer that as a cluster-level service (aka, this
would become part of the operate-first platform offer) ? 

I don't mean technically, but would that work with the operate-first
organization ?

**Comment by harshad16** [comment]
Yes correct, operate-first platform can provide these cluster-scoped resources.

**Comment by VannTen** [comment]
So to resume the discussion yesterday, although operate-first could spin
up an argo-workflow controller as a cluster service, we deploy to other
environments (I didn't know that ^) where it would be more complicated.
